xapian-bindings/configure.ac

```diff
@@ -94,6 +94,21 @@ XO_LIB_XAPIAN([], [],
 			  [^\([0-9]*\.[0-9]*[13579]\)\..*$], [-\1]))
 XO_REQUIRE([1.4.0])
 
+AC_ARG_ENABLE([letor],
+  [AS_HELP_STRING([--enable-letor], [enable make rules to build bindings related to xapian-letor])],
+  [case ${enableval} in
+    yes|no) ;;
+    *) AC_MSG_ERROR([bad value ${enableval} for --enable-letor]) ;;
+  esac],
+  [enable_letor=no])
+
+AM_CONDITIONAL([LETOR_FOUND], [test yes = "$enable_letor"])
```

Comment:
`LETOR_FOUND` isn't a great choice for this variable, given at this point it's "just" about whether it's requested or not. `BUILD_LETOR` or `BUILD_LETOR_BINDINGS` would be clearer.

Reply:
fixed in a fixup commit.